tc/sgx/trusted_worker_manager/enclave/enclave_utils.cpp

```diff
@@ -41,9 +41,9 @@ void Log(int level, const char* fmt, ...) {
     va_start(ap, fmt);
     vsnprintf_s(buf, BUFSIZ, fmt, ap);
     va_end(ap);
-#ifdef TCF_DEBUG_BUILD
+//#ifdef TCF_DEBUG_BUILD
```

Comment:
Do we want this for debug builds or not?

Reply:
No, this was added for debugging purpose. Accidently got committed, will remove it.